.env-template

```diff
@@ -1,4 +1,12 @@
 API_REF=
-MONGO_URI=
 API_PORT=
+
```

Comment:
```suggestion
```

Reply:
if we need to break this down into sections for API and Frontend vars I suggest we do so with comment headers like we did in the docker compose.

Reply:
Added headers. Is your suggestion to remove the blank line at the end of the file?